tests/foreman/ui/test_myaccount.py

```diff
@@ -111,9 +86,14 @@ def test_positive_update_firstname(self):
 
         :CaseImportance: Critical
         """
-        valid_strs = _valid_string_data()
-        valid_strs.append('name with space')
-        self.assert_text_field_update('users.firstname', *valid_strs)
+        valid_names = _valid_string_data()
+        valid_names.append('name with space')
+        for firstname in valid_names:
+            with self.subTest(firstname):
+                with self.session_test_user():
+                    element = self.my_account.update_firstname(firstname)
```

Comment:
first_name_element will be more readable, this apply to others elements bellow except the lang one 
perhaps should be selected_lang_element, what do you think?